lib/LDA.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+import numpy
+from lib.content_based import ContentBased
+from sklearn.decomposition import LatentDirichletAllocation
+from sklearn.feature_extraction.text import CountVectorizer
+
+
+class LDARecommender(ContentBased):
```

Comment:
I don't see why this class is called LDARecommender, if it does provide recommendation then it should extend abstract_recommender as well. if it does not then simply rename it to LDA.